docs/src/equations.md

```diff
@@ -173,8 +176,8 @@ The ``\nabla \cdot((\rho e_\text{int} + p) \boldsymbol{u}`` term is discretized
 ```math
 D_h[ (\rho e_\text{int} + p) (\boldsymbol{u}_h + I^c(\boldsymbol{u}_v))] +
 D^c_v\left[
-  I^f(\rho) U^f\left(I^f(\boldsymbol{u}_v, \frac{\rho e_\text{int} + p}{\rho} \right)
-  + (\rho e_\text{int} + p)  I^f(\boldsymbol{u}_h)
+  WI^f(J, \rho) U^f\left(\boldsymbol{u}_v, \frac{\rho e_\text{int} + p}{\rho} \right)
```

Comment:
Do we use the upwinding by default? I thought we weren't for energy?

Reply:
Not sure: @szy21?

Reply:
Default is no upwinding

Reply:
That's correct, the default is no upwinding.